test/pylib/random_tables.py

```diff
@@ -240,6 +240,8 @@ def __init__(self, test_name: str, cql: CassandraSession, keyspace: str):
         self.keyspace = keyspace
```

Comment:
Comments regarding the commit message:

1. "creation and drop should happen only once per cluster and no test should be running changes (either not started or finished)"

I had to think for a few minutes to interpret this.

Is this the corrent understanding: "creation and drop **of the keyspace** should happen only once per cluster and no test should be running changes (either not started or finished) **while the keyspace is being created or dropped**"?

2. All topology tests are for Scylla with Raft. So don't use the Cassandra
this_dc workaround as it's unnecessary for Scylla.

Nowhere in this patch we can find the string `this_dc`. If you didn't mention it in the commit message, I wouldn't even know that there's a problem with Cassandra and we needed some workaround for Cassandra at some point (why should I even care?)

And now I have to ask: **what** is the `this_dc` workaround? Sorry.